client/data/emails/use-remove-titan-mailbox-mutation.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * External dependencies
+ */
+import { useCallback } from 'react';
+import { useMutation, useQueryClient } from 'react-query';
+
+/**
+ * Internal dependencies
+ */
+import { queryKeyPrefix } from './use-emails-query';
+import wp from 'calypso/lib/wp';
+
+const noop = () => {};
+
+/**
+ * Deletes a mailbox from a Professional Email (Titan) account
+ *
+ * @param {string} domainName The domain name of the mailbox
+ * @param {string} mailboxName The mailbox name
+ * @param {object} mutationOptions Mutation options passed on to `useMutation`
+ * @returns {{ data, error, isLoading: boolean, removeTitanMailbox: Function, ...}} Returns various parameters piped from `useMutation`
+ */
+export function useRemoveTitanMailboxMutation( domainName, mailboxName, mutationOptions = null ) {
```

Comment:
Why do we have that `mutationOptions` parameter? We never pass such parameter when we call that function, so we could probably remove it to keep our code simple (we can still add it back whenever it is really needed).

Reply:
Please take another look. The options are being used.